app/javascript/packages/address-search/components/post-office-no-results.tsx

```diff
@@ -0,0 +1,23 @@
+import { getAssetPath } from '@18f/identity-assets';
+import { t } from '@18f/identity-i18n';
+
+function PostOfficeNoResults({ address }) {
+  return (
+    <>
+      <img
+        className="grid-col-2 inline-block veritcal-align-top margin-top-22"
```

Comment:
update spelling to `vertical`

Reply:
Good find! Resolved with commit 8ea92305750c22faa653bb9091562262ca2f13ed
Here is a screenshot with that change

![Screenshot 2023-09-05 at 3 35 52 PM](https://github.com/18F/identity-idp/assets/125507397/464e9008-1c0f-4c5b-835e-85f3081f96ac)